src/js/directives/theme-picker.js

```diff
@@ -0,0 +1,110 @@
+/**
+* @ngdoc directive
+* @name themePicker
+* @module enplug.sdk.utils
+*
+* @param selectedTheme {Object selected theme}
+* @param selectedTheme {Object default or new theme to use as templatee}
+* @param themeDefinition {Object constant of theme sections}
+* @param customThemes {Array of saved custom themes}
+* @param enplugThemes {Array of default enplug themes}
+* @param asset {Object current asset}
+*/
+angular.module('enplug.sdk.utils').directive('themePicker', function ($document, $enplugDashboard, $enplugAccount, gettextCatalog, $filter, $route) {
+    return {
+        restrict: 'E',
+        transclude: true,
+        scope: {
+            selectedTheme: '=',
+            defaultTheme: '=',
+            customThemes: '=',
+            enplugThemes: '=',
+            themeDefinition: '=',
+            appUrl: '=',
```

Comment:
I'd call it something like previewAppUrl and previewAsset, so it's clear what they are used for